pages/instructorAssessmentCheatDetection/cheat-detection-py/fa20E2cleanedOutput/assessment_instance_11552_log.csv

```diff
@@ -0,0 +1,145 @@
+,Unnamed: 0,Random_Id,data,date_iso8601,event_color,event_date,event_name,formatted_date,instance_question_id,instructor_question_number,qid,question_id,student_question_number,submission_id,variant_id,variant_number
```

Comment:
None of these files should be committed to the repo.

Reply:
You're right! This is where we want to shorten the workflow. Currently, we pull students' log data using an endpoint and run it inside of the repo. We are drafting a button to trigger similarity calculation, which firstly pulls students' log directly from db, and then writes the similarity scores into a new table in the db / csv file. Would this be a feasible design?

Reply:
That seems like a reasonable path to an MVP!